syncstorage/storage/sql/queries_generic.py

```diff
@@ -92,6 +93,47 @@
 DELETE_ITEMS = "DELETE FROM %(bso)s WHERE userid=:userid "\
                "AND collection=:collectionid AND id IN %(ids)s"
 
+CREATE_TRANSACTION = "INSERT INTO batch_uploads (batch, userid, collection) "\
+                     "VALUES (:batch, :userid, :collection)"
+
+VALID_TRANSACTION = "SELECT open FROM batch_uploads WHERE batch = :batch " \
+                    "AND userid = :userid AND collection = collection"
```

Comment:
This isn't used anywhere yet.

In general, what's the advantage of setting `open=0` as opposed to e.g. just deleting the entry from `batch_uploads` once we're done with it?